cordformation/src/main/kotlin/net/corda/plugins/Dockerform.kt

```diff
@@ -69,12 +97,44 @@ open class Dockerform @Inject constructor(objects: ObjectFactory) : Baseform(obj
         bootstrapNetwork()
         nodes.forEach(Node::buildDocker)
 
+        var services = mutableMapOf<String, Map<String, Any>>()
+
+        nodes.forEachIndexed {index, it ->
 
-        // Transform nodes path the absolute ones
-        val services = nodes.map {
             val nodeBuildDir = directoryPath.resolve(it.nodeDir.name).toAbsolutePath().toString()
+            var dependsOnList = mutableListOf<String>()
+
+            if (it.drivers.isNullOrEmpty() && project.hasProperty(DEFAULT_DB_GRADLE_PROPERTY)) {
+
+                it.dockerImage("entdocker.software.r3.com/corda-enterprise-java1.8-${it.runtimeVersion().toLowerCase()}")
+                it.dbSettings(DEFAULT_DB_STARTING_PORT + index, "${it.containerName}-db")
+                it.installDefaultDatabaseConfig(
+                        createDefaultDBConfig(it.dbSettings.port, it.dbSettings.host), DEFAULT_DB_INIT_FILE
+                )
+                val driverVersion = project.property(DEFAULT_DB_GRADLE_PROPERTY)
+                it.installDriver("org.postgresql", "postgresql", driverVersion)
```

Comment:
Where does this get the postgres driver from? Do we specify a repository, do we ask the user for one, or do we rely on the `build.gradle` repositories block?

Reply:
We rely on the repositories specified in the build.gradle block

Reply:
Yes, CorDapp developer will expect to use the `repositories` block in `build.gradle` to tell Gradle where to download everything from. I think they'll also expect to specify more than just its version though.